src/vos/evtree.c

```diff
@@ -966,7 +966,7 @@ evt_node_entry_free(struct evt_context *tcx, struct evt_node_entry *ne)
 
 	desc = evt_off2desc(tcx, ne->ne_child);
 	vos_dtx_degister_record(evt_umm(tcx), desc->dc_dtx,
-				umem_ptr2id(evt_umm(tcx), desc), DTX_RT_EVT);
+				umem_ptr2off(evt_umm(tcx), desc), DTX_RT_EVT);
```

Comment:
ne->ne_child would suffice here.   The desc pointer is set from that offset at 967

Reply:
OK, I will fix it.